nucliadb_protos/knowledgebox.proto

```diff
@@ -180,6 +181,23 @@ message VectorSets {
 }
 
 
+// Configuration values for a vectorset
+message VectorSetConfig {
+    string vectorset_id = 1;
+
+    nodewriter.VectorIndexConfig index_config = 2;
+
+    // list of possible subdivisions of the matryoshka embeddings (if the model
+    // supports it)
+    repeated uint32 matryoshka_dimensions = 3;
```

Comment:
shoudn't this be inside `VectorIndexConfig` ?

Reply:
No, `VectorIndexConfig` is only for the index configuration and it's not aware of the matryoshka concept